manifests/kustomize/base/installs/multi-user/pipelines-profile-controller/deployment.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     spec:
       containers:
       - name: profile-controller
-        image: python:3.9
+        image: public.ecr.aws/docker/library/python:3.12
```

Comment:
This one here is the most important change for end users. I am running this for a long time on 3.12